File: 250401/더 긴 문자열/longer-string.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
    string str1;
    string str2;
    cin >> str1 >> str2;

    if(str1.length() > str2.length())
        cout << str1 << " " << str1.length() << endl;
    else if (str1.length() < str2.length())
        cout << str2 << " " << str2.length() << endl;
    else 
        cout << "same" << endl;
    return 0;
}

File: 260212/더 긴 문자열/longer-string.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);

        String str = in.next();
        String str2 = in.next();

        if(str.length() > str2.length()){
            System.out.print(str + " " + str.length());
        }
        else if(str.length() == str2.length()){
            System.out.print("same");
        }
        else{
            System.out.print(str2 + " " + str2.length());
        }
    }
}
